lib/action.js

```diff
@@ -245,6 +245,28 @@ function action(system) {
 
 	});
 
+	// If a user wants to abort a single button actions
+	self.system.on('action_abort_one', function(bank) {
+
+		var bid = bank[0] + '_' + bank[1];
+		var b;
+		var c = 0;
```

Comment:
It would be nice to have a more descriptive name for `c` (eg `changed`)

Reply:
I'll change that to `cleared`